main.py

```diff
@@ -3,14 +3,31 @@
 # throughout this file
 import pygame
 
+from constants import SCREEN_HEIGHT, SCREEN_WIDTH
+
 
 def main():
-    try:
-        pygame.init()
-        print('Starting asteroids!')
-        pygame.quit()
-    except Exception as e:
-        print(f'Failed to initialize Pygame: {e}')
+    pygame.init()  # Initialize Pygame
+
+    # Set up the display
+    screen = pygame.display.set_mode((SCREEN_WIDTH, SCREEN_HEIGHT))
+    
+    # Start the game loop
+    clock = pygame.time.Clock()
+    while True:
+        clock.tick(60) 
+        # Event handling to check if the user wants to quit
+        for event in pygame.event.get():
+            if event.type == pygame.QUIT:
+                pygame.quit()
+                return  # Exit the game loop if the user closes the window
+
+        # Fill the screen with black color (RGB: 0, 0, 0)
+        screen.fill((0, 0, 0))
+
+        # Update the display
+        pygame.display.flip()
+        pygame.time.delay(10)  # Add a small delay to reduce CPU usage
```

Comment:
**Suggestion:** Remove unnecessary delay in the game loop.

**Existing Code:**
```Python
pygame.time.delay(10)  # Add a small delay to reduce CPU usage
```

**Improved Code:**
```Python
# Remove the delay to let clock.tick(60) manage the frame rate
```

**Details:**
Remove the `pygame.time.delay(10)` to avoid unnecessary delays in the game loop.